content/en/docs/tasks/security/cert-management/isito-csr/index.md

```diff
@@ -0,0 +1,284 @@
+---
+title: External Certificates using cert-manager
+description: Shows how to use a Custom Certificate Authority via cert-manager to provision Istio workload certificates.
+weight: 80
```

Comment:
The 80 matches the weight in the Plug in CA Certificates. You'll need to tweak this to a higher/lower value depending on if this should show up before or after Plug in CA Certificates on the Certificate Management page.